content/influxdb/v2.0/reference/flux/stdlib/testing/load.md

```diff
@@ -0,0 +1,49 @@
+---
+title: testing.load() function
+description: >
+  The `testing.load()` function loads tests data from a raw stream of tables.
+menu:
+  influxdb_2_0_ref:
+    name: testing.load
+    parent: Testing
+weight: 301
+introduced: 0.112.0
+---
+
+The `testing.load()` function loads tests data from a raw stream of tables.
```

Comment:
Any distinction between a raw stream of tables and a stream of tables? Data that hasn't been processed or calculated through other functions?

Reply:
Removed the word "raw."